packages/SwingSet/src/vats/network/network.js

```diff
@@ -508,7 +508,7 @@ export function makeNetworkProtocol(protocolHandler) {
   });
 
   // Wire up the local protocol to the handler.
-  E(protocolHandler).onCreate(protocolImpl, protocolHandler);
+  void E(protocolHandler).onCreate(protocolImpl, protocolHandler);
```

Comment:
@michaelfig is this right? Do we need to be more careful with a rejection from `onCreate` ?

Reply:
No, making it an unhandled rejection is perfectly fine.